mm2src/coins/utxo.rs

```diff
@@ -1625,9 +1626,16 @@ fn kmd_interest(
     if height >= 1_000_000 && minutes > 31 * 24 * 60 {
         minutes = 31 * 24 * 60;
     }
-    // next 2 lines ported as is from Komodo codebase
+    // next lines ported as is from Komodo codebase
     minutes -= 59;
-    let accrued = (value / 10_512_000) * minutes;
+    let mut accrued = (value / 10_512_000) * minutes;
```

Comment:
We can use constants for this value as it took me some time to understand what this number represents. Something like:

```rust
const MINUTES_PER_YEAR: u64 = 525_600;
const AUR_PER_MINUTE: f64 = 0.05 / MINUTES_PER_YEAR as f64;
let mut accrued = (value as f64 * AUR_PER_MINUTE) as u64 * minutes;
```
MINUTES_PER_YEAR can also be used in these places:

https://github.com/KomodoPlatform/atomicDEX-API/blob/6a0820a425d64590ab2d49ee345628ae0272b37a/mm2src/coins/utxo.rs#L1622-L1624

Reply:
> We can use constants for this value as it took me some time to understand what this number represents. Something like:

I kept these parts as is to make the code the same as the one in the komodo core repo https://github.com/DeckerSU/komodo/blob/64efc07200d9f34a76045927b631464698c3e30d/src/komodo_interest.cpp#L34-L36 as I thought it will make it easier to keep track of changes there. But I agree with you @caglaryucekaya using constants and `AUR_PER_MINUTE` makes the code clearer, so I will go with you suggestion.